tests/test_jx/test_set_ops.py

```diff
@@ -836,7 +836,7 @@ def test_select_leaves(self):
             "expecting_list": {
                 "meta": {"format": "list"},
                 "data": [
-                    {"a": {"b": "x", "v": 2, "c": {"z": 0}}},
+                    {"a": {"b": "x", "v": 2,}},
```

Comment:
I am glad to see you are willing to change the test cases!  This shows a desire to understand what should be expected.  Good work!